internal/indexmanagement/scripts.go

```diff
@@ -131,6 +131,45 @@ def updateWriteIndex(currentIndex, nextIndex, alias):
     print(e)
     sys.stdout = original_stdout
     return False
+
+def getCurrentDiskPercent():
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    response = es_client.cat.allocation(format="JSON", h="disk.percent,node", v=True)
+    diskPercent = []
+    for i in range(len(response)):
+      diskPercent.append(int(response[i]["disk.percent"]))
+    return max(diskPercent)
```

Comment:
I believe the approach by calculating the max over the collected disk space usage per node can be simplified by asking ES through the nodes stats API for the FS information: https://www.elastic.co/guide/en/elasticsearch/reference/6.8/cluster-nodes-stats.html#fs-info

Reply:
I do not see how it can be simplified, the stats API that you linked gives the following info for each node, here is a snapshot from my cluster:

> "fs" : {
        "timestamp" : 1646828572706,
        "total" : {
          "total_in_bytes" : 128300593152,
          "free_in_bytes" : 115112910848,
          "available_in_bytes" : 115112910848
        },
when I calculate the disk percentage using those values, I get the same result that the cat allocation API gives me without any calculations.
I end up with 3 values in both APIs, one for each node.

Reply:
@btaani You can ignore my comment. I accidentally thought that the `nodes/stats/fs` call will result in a total consumption output across all nodes.